packages/perseus/src/widgets/radio/cross-out-button.jsx

```diff
@@ -98,10 +98,12 @@ export default class CrossOutButton extends React.PureComponent<CrossOutButtonPr
                     //     bit fragile, but for now it seems better than adding
                     //     complexity to the ChoiceIcon API!
                     primaryProductColor={buttonAndTextColor}
+                    focused={false}
+                    hovered={false}
                     pressed={true}
                     checked={false}
-                    focused={false}
                     correct={false}
+                    multipleSelect={false}
```

Comment:
**suggsetion:** It seems like people should be able to cross-out choices when `multipleSelect` is true as well.  If that's the case, can you add a story for that situation?